styles/main.css

```diff
@@ -824,3 +824,11 @@ button {
   height: 50px;
   width: 50px;
 }
+
```

Comment:
Can you place new message styles below the other message styles. Thanks! Otherwise it becomes too hard to find.